com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/variables/VariableHandling.java

```diff
@@ -0,0 +1,24 @@
+package com.sap.cloud.lm.sl.cf.process.variables;
+
+import org.flowable.common.engine.api.variable.VariableContainer;
+
+public final class VariableHandling {
+
+    private VariableHandling() {
+    }
+
+    public static <T> void set(VariableContainer container, Variable<T> variable, T value) {
```

Comment:
How null values are handled? Will there be a problem if a null value is provided?

Reply:
Null values are handled in the serializers currently. I'll think about moving the handling here, though, as it seems like a good idea.